src/main/resources/static/js/projects/projects-controller.js

```diff
@@ -10,4 +10,9 @@ angular.module('CapstoneSystem')
             var data = table.row( this ).data();
             window.location.href = '/project';
         } );
+        $scope.myHtml = "<h5>Project Description...</h5>"

```

Comment:
can we make this a place holder of the text area?

Reply:
Yes, that is how it behaves now.

Reply:
Hmm I think it is a text on the text area now. A placeholder disappears when we focus on the input area. We can leave it as is now. I originally thought you wanted to make it a placeholder :smile

Reply:
Sounds good, I think once we persist the text area content it will save as myHtml